src/Nordea/Components/NumberInput.elm

```diff
@@ -127,8 +127,8 @@ getAttributes : Config msg -> List (Attribute msg)
 getAttributes config =
     let
         format value =
-            if value == "" then
-                ""
+            if String.endsWith "." value || String.endsWith "," value then
+                value
```

Comment:
```
                if value == "" then ""
```
This looks redundant because value = "" would return "" as well if it goes into the `else` track

Reply:
I guess it avoids the toFloat computation that is guaranteed failure?

Reply:
Yeah, i think so. The purpose might be to avoid that. But it actually does not matter because `else` track also handle well the situation if toFloat fails